spec/lib/checkingaccount_spec.rb

```diff
@@ -0,0 +1,169 @@
+describe BankAccount::CheckingAccount do
```

Comment:
Note: If you wanted to avoid having to say `BankAccount::CheckingAccount` every time in your tests, you can wrap the entire file in `module BankAccount` (the same way you wrapped the class you're testing) and everything should be properly namespaced.